core/domain/user_services.py

```diff
@@ -888,16 +1025,22 @@ def _save_existing_users_settings(user_settings_list):
     user_ids = [user.user_id for user in user_settings_list]
     user_settings_models = user_models.UserSettingsModel.get_multi(
         user_ids, include_deleted=True)
+    strict_users_model = []
```

Comment:
Ditto.

Reply:
Variable names are changed, Acc. to the new discussion!